### PR TITLE
Wo/send message for feeds

### PR DESCRIPTION
**Pull request by wogara**
For slack application, we need to add the slack webhook endpoint as the request URL under "Interactivity & Shortcuts." Also need to give chat:write.public permissions for the slack bot.

### CODE REVIEW

File: server-phoenix/lib/helios_web/controllers/web_hooks/slack_controller.ex [hunk 1]
```diff
@@ -38,17 +38,47 @@ defmodule HeliosWeb.WebHooks.SlackController do
     end
   end
 
-  def handle(conn, params) do
```
Comment (chewnoill) [inline server-phoenix/lib/helios_web/controllers/web_hooks/slack_controller.ex:40]: I think this got removed from the wrong commit?

File: server-phoenix/mix.exs [hunk 2]
```diff
@@ -66,7 +66,8 @@ defmodule Helios.MixProject do
       {:quantum, "~> 3.0"},
       {:elixir_auth_google, "~> 1.6.3"},
       {:mock, "~> 0.3.0", only: :test},
-      {:json, "~> 1.4"}
+      {:json, "~> 1.4"},
+      {:torch, "~> 4.2.1"}
```
Comment (chewnoill) [inline server-phoenix/mix.exs:70]: I think this got merged wrong

**Review by cathyzoller — APPROVED** [review]
Everything seems to be in order.  Great work Billy.

**Comment by sethkrasnianski** [comment]
@wogara did you mean to merge master into your feature branch instead of rebasing?

**Comment by wogara** [comment]
> @wogara did you mean to merge master into your feature branch instead of rebasing?

Ah, i think you're right. should've rebased.

**Comment by chewnoill** [comment]
@sethkrasnianski @wogara merge commits get dropped automatically when the PR is "merged" (actually a rebase forced from project settings) to master

maybe a little confusing though.
cc @cbarber

**Pull request merged**